packages/apollo-gateway/src/snapshotSerializers/queryPlanSerializer.ts

```diff
@@ -62,7 +65,17 @@ function printNode(
             indentationNext
           : '') +
         printer(
-          node.selectionSet,
+          visit(parse(node.operation), {
+            OperationDefinition: op => {
+              if(isQueryPlanOperation(op)) {
```

Comment:
I don't quite understand what a "query plan operation" (an unnamed query that has `representations` as a variable?) or why it's special cased here.  Can you help me understand that?

Reply:
Can we rename this predicate to `isEntitiesOperation`?

Reply:
Renamed (in next commit) 👍 @abernix This is to preserve the printing we do in the query plan printer right now. For all the fetch nodes to entities, we don't print the `_entities` fetch explicitly or the `representations` variable declarations. 

For example, on a fetchNode for a query plan, without removing this definition, this is the difference in what would be printed:

```diff
# with the node removed
- {
-   ... on User {
-     risk
-   }
- }

# without the node removed
+ {
+ _entities(representations: $representations) {
+   ... on User {
+     risk
+   }
+ }
```

Reply:
Thank you for the clarification, @JakeDawkins.